public/app/percona/shared/helpers/permissions.ts

```diff
@@ -0,0 +1,3 @@
+import { OrgRole } from 'app/types';
+
+export const isPmmAdmin = (user: any): boolean => user.isGrafanaAdmin || user.orgRole === OrgRole.Admin;
```

Comment:
The User has a type, so we can use it `import { User } from 'app/core/services/context_srv';`

Reply:
Thanks, didn't see that one 👍